Remotion/Web/Development.WebTesting/IControlHost.cs

```diff
@@ -32,8 +31,8 @@ public interface IControlHost
     /// <typeparam name="TControlObject">The type of the child control to be found.</typeparam>
     /// <param name="controlSelectionCommand">Specifies the child control, see <see cref="IControlSelectionCommand{TControlObject}"/> for more information.</param>
     /// <returns>The specified child <see cref="ControlObject"/>.</returns>
-    /// <exception cref="AmbiguousException">If the selection command cannot unambiguously identify the child control.</exception>
-    /// <exception cref="MissingHtmlException">If the selection command cannot find the child control.</exception>
+    /// <exception cref="WebTestException">If the selection command cannot unambiguously identify the child control.</exception>

```

Comment:
We should probably merge this into a single Exception doc

Reply:
We found docs using this style.